index.html

```diff
@@ -0,0 +1,200 @@
+                                    Forum
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Listen to speakers from various countries about the messeges of sharing and opening.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="fas fa-praying-hands"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    Workshop
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Try creating your own work using open source license rather than just watching.
+                                </p>
+                            </div>
+                        </li>
+                        <li class="program-list-item">
+                            <div class="program-list-item-container">
+                                <div class="program-list-item-container__icon">
+                                    <i class="fas fa-project-diagram"></i>
+                                </div>
+                                <h2 class="program-list-item-container__title">
+                                    DMC Ignite
+                                </h2>
+                                <p class="program-list-item-container__text">
+                                    Get opporunites to network with DMC affiliates around the world, also after the summit.
+                                </p>
+                            </div>
+                        </li>
+                    </ul>
+                    <button class="btn btn-large" type="button">SEE THE WHOLE PROGRAM</button>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+              <div class="speakers">
+                  <h2 class="speakers-heading">Featured Speakers</h2>
+                    <ul class="speakers-list" id="speakers-list">
+                        
+                    </ul>
+               </div>
+          </div>
+        </section>
+        <section>
+            <div class="container bg-t">
+                <div class="partner">
+                    <h2 class="partner-heading">Partner</h2>
+                    <ul class="partners-list">
+                        <li class="partner-list-item">
+                            <i class="fab fa-facebook"></i>
+                        </li>
+                        <li class="partner-list-item">
```

Comment:
- [ ] Kindly include the images in the partners section as seen in the original.
- [ ] Also, ensure you use the exact background color, you can use a color picker extension to get the right color.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/128994518-5010fd42-36c7-417d-8929-961706e57613.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/128994584-37968658-5911-40c7-be39-163f390e9852.png' height='250px'> |